### PR TITLE
DOCSUP-14103: Edit and translate to Russian (OpenSky dataset and UK Property Price Paid data)

### PR DESCRIPTION
**Pull request by ka1bi4**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Changelog category (leave one):
- Documentation for #27437, #27432 and #27479.

### CODE REVIEW

File: docs/ru/getting-started/example-datasets/opensky.md [hunk 1]
```diff
@@ -0,0 +1,421 @@
...
+
+## Загрузите набор данных
+
+Выполните команду:
+
+```bash
+wget -O- https://zenodo.org/record/5092942 | grep -oP 'https://zenodo.org/record/5092942/files/flightlist_\d+_\d+\.csv\.gz' | xargs wget
+```
+
+Загрузка займет около 2 минут при хорошем подключении к Интернету. Будет загружено 30 файлов, общим размером 4,3 ГБ.
```
Comment (olgarev) [inline docs/ru/getting-started/example-datasets/opensky.md:25]:
```suggestion
Загрузка займет около 2 минут при хорошем подключении к интернету. Будет загружено 30 файлов общим размером 4,3 ГБ.
```

File: docs/ru/getting-started/example-datasets/opensky.md [hunk 2]
```diff
@@ -0,0 +1,421 @@
...
+    altitude_1 Float64,
+    latitude_2 Float64,
+    longitude_2 Float64,
+    altitude_2 Float64
+) ENGINE = MergeTree ORDER BY (origin, destination, callsign);
+```
+
+## Импортируйте данные в ClickHouse
+
+Загрузите параллельными потоками данные в ClickHouse:
```
Comment (olgarev) [inline docs/ru/getting-started/example-datasets/opensky.md:53]:
```suggestion
Загрузите данные в ClickHouse параллельными потоками:
```

File: docs/ru/getting-started/example-datasets/opensky.md [hunk 3]
```diff
@@ -0,0 +1,421 @@
...
+
+```bash
+ls -1 flightlist_*.csv.gz | xargs -P100 -I{} bash -c 'gzip -c -d "{}" | clickhouse-client --date_time_input_format best_effort --query "INSERT INTO opensky FORMAT CSVWithNames"'
+```
+
+- Список файлов передаётся (`ls -1 flightlist_*.csv.gz`) в `xargs` для параллельной обработки.
+- `xargs -P100` указывает на использование до 100 параллельных обработчиков, но поскольку у нас всего 30 файлов, то количество обработчиков будет всего 30.
```
Comment (olgarev) [inline docs/ru/getting-started/example-datasets/opensky.md:60]:
```suggestion
- `xargs -P100` указывает на возможность использования до 100 параллельных обработчиков, но поскольку у нас всего 30 файлов, то количество обработчиков будет всего 30.
```

File: docs/ru/getting-started/example-datasets/opensky.md [hunk 4]
```diff
@@ -0,0 +1,421 @@
...
+- Для каждого файла `xargs` будет запускать скрипт с `bash -c`. Сценарий имеет подстановку в виде ` {}`, а команда `xargs` заменяет имя файла (мы указали это для xargs с помощью `-I{}`).
```
Comment (olgarev) [inline docs/ru/getting-started/example-datasets/opensky.md:61]:
```suggestion
- Для каждого файла `xargs` будет запускать скрипт с `bash -c`. Сценарий имеет подстановку в виде `{}`, а команда `xargs` заменяет имя файла на указанные в подстановке символы (мы указали это для `xargs` с помощью `-I{}`).
```

File: docs/ru/getting-started/example-datasets/opensky.md [hunk 5]
```diff
@@ -0,0 +1,421 @@
...
+- Скрипт распакует файл (`gzip -c -d "{}"`) в стандартный вывод (параметр`-c`) и перенаправит его в `clickhouse-client`.
```
Comment (olgarev) [inline docs/ru/getting-started/example-datasets/opensky.md:62]:
```suggestion
- Скрипт распакует файл (`gzip -c -d "{}"`) в стандартный вывод (параметр `-c`) и перенаправит его в `clickhouse-client`.
```

File: docs/en/getting-started/example-datasets/opensky.md [hunk 6]
```diff
@@ -50,65 +52,97 @@ CREATE TABLE opensky
...
+```bash
+ls -1 flightlist_*.csv.gz | xargs -P100 -I{} bash -c 'gzip -c -d "{}" | clickhouse-client --date_time_input_format best_effort --query "INSERT INTO opensky FORMAT CSVWithNames"'
 ```
 
-Here we pass the list of files (`ls -1 flightlist_*.csv.gz`) to `xargs` for parallel processing.
+- Here we pass the list of files (`ls -1 flightlist_*.csv.gz`) to `xargs` for parallel processing.
 `xargs -P100` specifies to use up to 100 parallel workers but as we only have 30 files, the number of workers will be only 30.
+- For every file, `xargs` will run a script with `bash -c`. The script has substitution in form of `{}` and the `xargs` command will substitute the filename to it (we have asked it for xargs with `-I{}`).
+- The script will decompress the file (`gzip -c -d "{}"`) to standard output (`-c` parameter) and the output is redirected to `clickhouse-client`.
+- We also asked to parse DateTime fields with extended parser (`--date_time_input_format best_effort`) to recognize ISO-8601 format with timezone offsets.
```
Comment (olgarev) [inline docs/en/getting-started/example-datasets/opensky.md:63]:
```suggestion
- For every file, `xargs` will run a script with `bash -c`. The script has substitution in form of `{}` and the `xargs` command will substitute the filename to it (we have asked it for `xargs` with `-I{}`).
- The script will decompress the file (`gzip -c -d "{}"`) to standard output (`-c` parameter) and the output is redirected to `clickhouse-client`.
- We also asked to parse `DateTime` fields with extended parser (`--date_time_input_format best_effort`) to recognize ISO-8601 format with timezone offsets.
```

File: docs/ru/getting-started/example-datasets/opensky.md [hunk 7]
```diff
@@ -0,0 +1,421 @@
...
+
+Результат:
+
+```text
+┌──count()─┐
+│ 66010819 │
+└──────────┘
+```
+
+Проверьте, размер набора данных в ClickHouse составляет всего 2,66 ГБ.
```
Comment (olgarev) [inline docs/ru/getting-started/example-datasets/opensky.md:92]:
```suggestion
Убедитесь, что размер набора данных в ClickHouse составляет всего 2,66 ГБ.
```

File: docs/ru/getting-started/example-datasets/opensky.md [hunk 8]
```diff
@@ -0,0 +1,421 @@
...
+129. │ 2021-06-14 │ 5613 │ ████████████████████████████████████████████████████████▏                    │
+130. │ 2021-06-21 │ 6061 │ ████████████████████████████████████████████████████████████▌                │
+131. │ 2021-06-28 │ 2554 │ █████████████████████████▌                                                   │
+     └────────────┴──────┴──────────────────────────────────────────────────────────────────────────────┘
+```
+
+### Online Playground
+
+Вы можете протестировать другие запросы к этому набору данным с помощью интерактивного ресурса [Online Playground](https://gh-api.clickhouse.tech/play?user=play). Например, [вот так](https://gh-api.clickhouse.tech/play?user=play#U0VMRUNUCiAgICBvcmlnaW4sCiAgICBjb3VudCgpLAogICAgcm91bmQoYXZnKGdlb0Rpc3RhbmNlKGxvbmdpdHVkZV8xLCBsYXRpdHVkZV8xLCBsb25naXR1ZGVfMiwgbGF0aXR1ZGVfMikpKSBBUyBkaXN0YW5jZSwKICAgIGJhcihkaXN0YW5jZSwgMCwgMTAwMDAwMDAsIDEwMCkgQVMgYmFyCkZST00gb3BlbnNreQpXSEVSRSBvcmlnaW4gIT0gJycKR1JPVVAgQlkgb3JpZ2luCk9SREVSIEJZIGNvdW50KCkgREVTQwpMSU1JVCAxMDA=). Однако, обратите внимание, что здесь нельзя создавать временные таблицы.
```
Comment (olgarev) [inline docs/ru/getting-started/example-datasets/opensky.md:421]:
```suggestion
Вы можете протестировать другие запросы к этому набору данным с помощью интерактивного ресурса [Online Playground](https://gh-api.clickhouse.tech/play?user=play). Например, [вот так](https://gh-api.clickhouse.tech/play?user=play#U0VMRUNUCiAgICBvcmlnaW4sCiAgICBjb3VudCgpLAogICAgcm91bmQoYXZnKGdlb0Rpc3RhbmNlKGxvbmdpdHVkZV8xLCBsYXRpdHVkZV8xLCBsb25naXR1ZGVfMiwgbGF0aXR1ZGVfMikpKSBBUyBkaXN0YW5jZSwKICAgIGJhcihkaXN0YW5jZSwgMCwgMTAwMDAwMDAsIDEwMCkgQVMgYmFyCkZST00gb3BlbnNreQpXSEVSRSBvcmlnaW4gIT0gJycKR1JPVVAgQlkgb3JpZ2luCk9SREVSIEJZIGNvdW50KCkgREVTQwpMSU1JVCAxMDA=). Однако обратите внимание, что здесь нельзя создавать временные таблицы.
```

File: docs/ru/getting-started/example-datasets/uk-price-paid.md [hunk 9]
```diff
@@ -0,0 +1,649 @@
+---
+toc_priority: 20
+toc_title: Набор данных о стоимости недвижимости в Великобритании
+---
+
+# Набор данных о стоимости недвижимости в Великобритании
+
+Набор данных содержит данные о стоимости недвижимости в Англии и Уэльсе. Данные доступны с 1995 года.
```
Comment (olgarev) [inline docs/ru/getting-started/example-datasets/uk-price-paid.md:8]:
```suggestion
Набор данных содержит информацию о стоимости недвижимости в Англии и Уэльсе. Данные доступны с 1995 года.
```

File: docs/ru/getting-started/example-datasets/uk-price-paid.md [hunk 10]
```diff
@@ -0,0 +1,649 @@
...
+Размер набора данных в несжатом виде составляет около 4 гигабайт, а в ClickHouse он займет около 278 Мбайт.
```
Comment (olgarev) [inline docs/ru/getting-started/example-datasets/uk-price-paid.md:9]:
```suggestion
Размер набора данных в несжатом виде составляет около 4 GiB, а в ClickHouse он займет около 278 MiB.
```

File: docs/ru/getting-started/example-datasets/uk-price-paid.md [hunk 11]
```diff
@@ -0,0 +1,649 @@
...
+
+## Загрузите набор данных
+
+Выполните команду:
+
+```bash
+wget http://prod.publicdata.landregistry.gov.uk.s3-website-eu-west-1.amazonaws.com/pp-complete.csv
+```
+
+Загрузка займет около 2 минут при хорошем подключении к Интернету.
```
Comment (olgarev) [inline docs/ru/getting-started/example-datasets/uk-price-paid.md:24]:
```suggestion
Загрузка займет около 2 минут при хорошем подключении к интернету.
```

File: docs/ru/getting-started/example-datasets/uk-price-paid.md [hunk 12]
```diff
@@ -0,0 +1,649 @@
...
+
+## Обработайте и импортируйте данные
+
+В этом примере используется `clickhouse-local` для предварительной обработки данных и `clickhouse-client` для импорта данных.
+
+Указывается структура исходных данных CSV-файла и запрос для предварительной обработки данных с помощью `clickhouse-local`.
+
+Предварительная обработка включает:
+- разделение почтового индекса на два разных столбца `postcode1` и `postcode2`, что лучше подходит для хранения данных и выполнения запросов к ним;
+- преобразование поля `time` в `Date`, поскольку оно содержит только время 00:00;
```
Comment (olgarev) [inline docs/ru/getting-started/example-datasets/uk-price-paid.md:57]:
```suggestion
- преобразование поля `time` в дату, поскольку оно содержит только время 00:00;
```

File: docs/ru/getting-started/example-datasets/uk-price-paid.md [hunk 13]
```diff
@@ -0,0 +1,649 @@
...
+- поле `uuid` игнорируется, потому что оно не будет использовано нами для анализа;
```
Comment (olgarev) [inline docs/ru/getting-started/example-datasets/uk-price-paid.md:58]:
```suggestion
- поле `uuid` игнорируется, потому что оно не будет использовано для анализа;
```

File: docs/ru/getting-started/example-datasets/uk-price-paid.md [hunk 14]
```diff
@@ -0,0 +1,649 @@
...
+
+Результат:
+
+```text
+┌──count()─┐
+│ 26321785 │
+└──────────┘
+```
+
+Размер набора данных в ClickHouse составляет всего 278 МБ, проверьте это.
```
Comment (olgarev) [inline docs/ru/getting-started/example-datasets/uk-price-paid.md:121]:
```suggestion
Размер набора данных в ClickHouse составляет всего 278 MiB, проверьте это.
```

**Review by alexey-milovidov — APPROVED** [review]

**Comment by ka1bi4** [comment]
I'll fix it tonight.

File: docs/en/getting-started/example-datasets/uk-price-paid.md [hunk 15]
```diff
@@ -346,67 +375,79 @@ ALTER TABLE uk_price_paid
...
-│ 2020 │ 368974 │ █████████████████████████████▌         │
-│ 2021 │ 384351 │ ██████████████████████████████▋        │
+│ 2014 │ 279915 │ ██████████████████████▍                │
+│ 2015 │ 297266 │ ███████████████████████▋               │
+│ 2016 │ 313201 │ █████████████████████████              │
+│ 2017 │ 346097 │ ███████████████████████████▋           │
+│ 2018 │ 350116 │ ████████████████████████████           │
+│ 2019 │ 351013 │ ████████████████████████████           │
+│ 2020 │ 369420 │ █████████████████████████████▌         │
+│ 2021 │ 386903 │ ██████████████████████████████▊        │
```
Comment (alexey-milovidov) [inline docs/en/getting-started/example-datasets/uk-price-paid.md:442]: Good, I see you've actually run these tutorials :)

**Review by alexey-milovidov — APPROVED** [review]

File: docs/en/getting-started/example-datasets/uk-price-paid.md (hunk 15 above)
Comment (ka1bi4) [reply docs/en/getting-started/example-datasets/uk-price-paid.md:442]: > Good, I see you've actually run these tutorials :)

Yeah, working with datasets is very engaging!

**Pull request merged**